src/style.css

```diff
@@ -1,3 +1,77 @@
+*{
+  box-sizing: border-box;
+}
+
+a{
+  text-decoration: none;
+}
+
+:root{
+  --blue-color: #00ACDC;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.header{
+  width: 100%;
+  display: flex;
+  align-items: center;
+  padding: 0 50px;
+  justify-content: space-between;
+}
+
+img{
```

Comment:
you should style class selectors instead of tag ones

Reply:
add a class selector to your `img` tag (for example `'logo-img'`) and then add styles to this selector

`.logo-img {`

Reply:
@Viktor-Kost 
![image](https://github.com/mate-academy/layout_moyo-header/assets/70914734/abf4d6b6-bcbf-4d9e-b8a8-fbc4cb784d3c)